isso/ext/notifications.py

```diff
@@ -104,9 +109,10 @@ def __iter__(self):
         yield "comments.new:after-save", self.notify_new
         yield "comments.activate", self.notify_activated
 
-    def format(self, thread, comment, parent_comment, recipient=None, admin=False):
+    def format(self, thread, comment, parent_comment, recipient=None):
 
         rv = io.StringIO()
+        admin = not parent_comment
```

Comment:
I don't understand this logic, why are we in admin mode if there is no parent comment?